tests/content_types_test.py

```diff
@@ -4,7 +4,10 @@
 
 from superdesk.tests import TestCase
 from unittest import mock
-from apps.content_types import content_types, apply_schema
+try:
+    from apps.content_types import content_types, apply_schema
+except ImportError:
```

Comment:
This is something which is makes sense to figure out why it happens..

Reply:
forgot to remove it actually